### DISCUSSION TITLE
南韓防疫追蹤系統

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#143] [General]
**新聞內容:**
南韓已經擁有高科技聯絡追蹤系統，主要拜信用卡紀錄、手機地點連網紀錄和監視器影像等個人資料所賜，根據富川市提交給南韓未來創造科學部的110頁商業資料，這套系統利用**AI演算法和臉部辨識技術**，分析逾10,820台監視器蒐集到的人群足跡資料，並且追蹤確診者、和染疫者近距離接觸者的足跡，並且分析是否有戴口罩。
中國大陸、俄羅斯、印度、波蘭、日本和美國幾個州都推出、或者試驗採用臉部辨識系統來追蹤新冠病毒病患。
該市有逾80萬人口，這套系統應該會協助減輕防疫追蹤團隊的壓力，明年1月將在首爾西部的富川市實行上述計畫。

**新聞心得:**
在人口那麼大的城市裡執行追蹤系統，的確能讓防疫團隊更能掌握感染者的足跡，雖然說在人口眾多的國家都有實行，可能防疫當前，人的隱私多少都先被冒犯到一些，幫助掌握情勢是一大優點，但侵犯隱私卻是一個問題。

**新聞來源:**
https://udn.com/news/story/6811/5957630

**Comment by qeeeee39** [comment]
AI越強大，越多個人隱私資訊被收集使用。
目前我還沒想到可以反這個論點的例子，不過我覺得造成這個現象的原因是來自人性，最原本的AI是中性不善不惡的。

**Reply by cobb110202008** [reply]
認同，任何的科技都像一把雙面刃，人性的善惡才是決定這項科技造成的後果。

**Comment by wh20502** [comment]
我覺得其實實名制也有這個問題，我不確定我現在接到的投資電話到底是我去哪裡簽名留下電話，慘遭外流，資安問題真的很嚴重。科技都像一把雙面刃，人性的善惡才是決定這項科技造成的後果。這句話我同意，但感覺有心人士總是濫用。

**Comment by lx3211** [comment]
感覺可以應用到緝捕通緝犯的搜尋網上，但一定會侵犯到隱私權了

**Comment by xCk27x** [comment]
有技術可以為圖片增加雜訊
防止AI蒐集你的人像資料
>關鍵字: Fawkes AI

**Reply by skyling0299** [reply]
之前去年會有看到一個分享，講說在圖片中加一些躁點就能讓ML的結果偏向，導到完全不同的方向。或許我們也能利用這個來防範辨識

**Comment by limitx0** [comment]
這感覺有點可怕
在不知不覺間自己的隱私都沒了

**Comment by Ben6uo** [comment]
太可怕了 如果被黑了的話怎麼辦

**Comment by HsuChen88** [comment]
如果資料被駭了，電影的畫面就要搬出螢幕了。

**Comment by larry40305** [comment]
中國表示：你們還要在乎隱私，我們都不用ㄟ

**Comment by ghost** [comment]
在科技發展的同時，也要注重隱私，而不是不管任何人的權益，應該要在科技以及隱私之間做平衡，社會才會越來越好。

**Comment by bww0401** [comment]
科技始於人性，最主要還是看我們怎麼去運用吧，但不可否定的是，現在資安問題真的很嚴重，如何在發展科技的同時，權衡個人隱私與全體利益將是一大課題

**Comment by yjpxi** [comment]
我覺得過度的監控可能會導致公眾對政府和相關機構的信任度下降，當人們感到自己的隱私被過度侵犯時，可能會引發反感和抵觸情緒，反而不利於防疫工作的開展。
